lib/puma/events.rb

```diff
@@ -97,6 +99,7 @@ def parse_error(server, env, error)
         "(#{env[HTTP_X_FORWARDED_FOR] || env[REMOTE_ADDR]}#{env[REQUEST_PATH]}): " \
```

Comment:
same here. No more puts. Only calls to the new interface, which should probably just end up calling `stderr.puts` by default.

Reply:
Done